no-literal-securerandom-exact-dljc-cmd.sh

```diff
@@ -1,29 +1,58 @@
 #!/bin/sh
 
-## This is an example "one-command with no arguments" script that controls run-dljc.sh. It is copied directly
-## from the experiments for the Continuous Compliance paper. If you want to re-use this script, you'll have to change
-## different things depending on what you're doing differently:
-# * If you are running on a non-CSE machine, change the definitions of JAVA11_HOME and JAVA8_HOME.
-# * EVERYONE must change the definitions of CHECKERFRAMEWORK and ANDROID_HOME. Setting ANDROID_HOME is optional.
-# * EVERYONE must change all instances of "securerandom" in this file to the name of the .list file containing your target repositories.
-# * If you are running a checker other than the no-literal checker, change the -c, -q, -l, and -s arguments to run-dljc.sh to make sense
-#   for the checker you are running.
-# * Make sure that all the absolute paths in this file match your machine/experimental directory.
+## This script runs run-dljc.sh with appropriate arguments and
+## environment variables. It is copied from the experiments for the
+## Continuous Compliance paper. If you want to re-use this script,
+## you'll have to change environment variables and paths below.  Note
+## that this script is intended for use with a custom typechecker
+## (i.e., a typechecker that is not in the main Checker Framework
+## distribution).  If your typechecker is in the main Checker
+## Framework distribution, you should use run-dljc.sh or
+## configure-and-exec-dljc.sh directly rather than making a copy of
+## this script.
+
+## Change these if you are running on a non-CSE machine.
 
 export JAVA11_HOME=/usr/lib/jvm/java-11-openjdk/
 export JAVA8_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+
+## Everyone must change these.
+
+export PARENTDIR=${HOME}/compliance-experiments/fse20
+export CHECKERFRAMEWORK=${PARENTDIR}/checker-framework
+checker=org.checkerframework.checker.noliteral.NoLiteralChecker
+checkername=no-literal
+repolist=securerandom.list
+
+# the stub files for the checker being used
+custom_stubs=/homes/gws/kelloggm/compliance-experiments/fse20/no-literal-checker/no-literal-checker/stubs
+
+# The qualifier classpath. In a custom checker, this is usually found
+# by running ./gradlew printClasspath in the mychecker-qual subproject.
```

Comment:
This is a helpful comment.  How about replacing the below line by a gradle invocation, such as `cd XXX && ./gradlew -q printClasspath`?  That will be less error-prone since it is automatically the latest version, and it's easier for others to reuse too.

Same comment about the next variable.

Reply:
Unfortunately, `printClasspath` is not a standard gradle target. I just opened a [PR](https://github.com/typetools/templatefora-checker/pull/30) to add it to the checker template, but there's no guarantee that any given checker will actually have it. The task is pretty simple, so I'll just copy its definition into this comment.